### ISSUE TITLE
Error in FUN(X[[i]], ...) :    Invalid comparison operator in dependency: >= 

### ISSUE THREAD

**Issue by karchjd** [krlmlr/r-appveyor#122]
My appveyor build fails even though building on travis, my machine and winbuilder is succesful.

I added appveyor support to my package using 

`devtools::use_appveyor()`

This created  [appveyor.yml](https://github.com/karchjd/gppm/blob/c1965cb03c3ee5a102243859725127acde98b79a/appveyor.yml)

The build on appveyor fails with the following error

```
package 'remotes' successfully unpacked and MD5 sums checked
The downloaded binary packages are in
	C:\Users\appveyor\AppData\Local\Temp\1\RtmpgN24M8\downloaded_packages
+ return
+ Rscript -e 'if (!("remotes" %in% rownames(installed.packages()))) q(status=1)'
+ Rscript -e 'options(repos=c(CRAN="https://cran.rstudio.com")); remotes::install_deps(dependencies = TRUE, type="both")'
Error in FUN(X[[i]], ...) : 
  Invalid comparison operator in dependency: >=  
Calls: <Anonymous> ... dev_package_deps -> local_package_deps -> lapply -> FUN
Execution halted
Command exited with code 1
7z a failure.zip *.Rcheck\*
```

As far as I know, >= is a valid comparison operator for dependencies. Might have to do with the following devtools issue? https://github.com/r-lib/devtools/issues/1247.

For completeness, [this is the package](https://github.com/karchjd/gppm/tree/c1965cb03c3ee5a102243859725127acde98b79a) and [this](https://github.com/krlmlr/r-appveyor/files/2123939/appout.txt) the full appveyor console output.

**Comment by rtobar** [comment]
I might be arriving a bit too late, but I wanted to drop my two cents.

I came across the same issue today. In the case of the package I was inspecting today was due to a trailing whitespace in one of the dependency lists. This is indeed related to the issue indicated above in devtools/remotes, and nothing to do with r-appveyor. The issue was easily avoidable by removing any trailing whitespaces.

In the case above, I think you have two trailing whitespaces just after the roxygen package spec (and one after "Suggest:".
